features/step_definitions/steps.rb

```diff
@@ -9,3 +9,15 @@
 Given(/^there is a dish named "([^"]*)" in our database$/) do |dish_name|
   Dish.create(name: dish_name)
 end
+
+Given(/^its dish price is "([^"]*)" in our database$/) do |dish_price|
+  Dish.create(price: dish_price)
+end
+
+Then(/^I should se a message saying "([^"]*)"$/) do |message|
+  expect(page).to have_content message
+end
+
+Given(/^its dish desription is "([^"]*)" in our database$/) do |dish_description|
+  Dish.create(description: dish_description)
```

Comment:
Okay, so you fixed the right attribute problem I was pointing to. All good. But, there is another problem. ;-) 
Here you are creating a totally new dish without a name and price but with a description. What you want to do is to make sure that your "Carbonara" dis has a description right? 

so what you need to do is:
1. Find the dish with name "Carbonara" in the database
2. Update its description with whatever value you have stored in the `dish_description` variable

Not create a totally new dish. Does that make sense?

Reply:
Something like `dish = Dish.first(:name => ‘Carbonara’)`? And `dish.update(:description => 'whatever this dish is')`?